File: FE/Basis.Store/src/app/private/catalogo/livros/features/listar-livros/dtos/ListarLivrosPaginadoRequest.ts
import { PaginacaoDto } from '../../../../../../common/dtos/Paginacao.dto';
import { LivroFilter } from '../../../models/LivroFilter';

export interface ListarLivrosPaginadoRequest {
    Paginacao: PaginacaoDto;
    Filtro: LivroFilter;
}

File: BE/Basis.Store/Basis.Store.Application/UseCases/Catalogo/Livros/Listar/Paginado/v1/DTOs/ListarLivrosPaginadoRequest.cs
using Basis.Store.Application.Common.Paginacao;

namespace Basis.Store.Application.UseCases.Catalogo.Livros.Listar.Paginado.v1.DTOs
{
    public record ListarLivrosPaginadoRequest
    {
        public PaginacaoDto Paginacao { get; set; } = new PaginacaoDto();
        public LivroFilter Filtro { get; set; } = new LivroFilter();
    }
}
